app/views/Project/Tagging/Sources/SourcesTable/index.tsx

```diff
@@ -342,38 +414,43 @@ function SourcesTable(props: Props) {
                 },
             ),
             publisherColumn,
-            createStringColumn<Lead, number>(
-                'authorsDetail',
+            createStringColumn<Lead, string>(
+                'authors',
                 _ts('sourcesTable', 'authors'),
-                (item) => item?.authorsDetail.map((v) => v.title).join(', '),
+                (item) => item.authors?.map((v: Pick<OrganizationType, 'title' | 'mergedAs'>) => {
+                    if (v.mergedAs) {
+                        return v.mergedAs.title;
+                    }
+                    return v.title;
+                }).join(', '),
```

Comment:
We already have a organization title selector in OrganizationSelectInput, let's use that